week-03-ajax-json-express/day-02-ajax/dusk-get-post/solutions.md

```diff
@@ -0,0 +1,16 @@
+## Solutions - AJAX: GET & POST
+
+
+#### Postman
+
+1. Use Postman to `GET` all the `books`. Now get just one book using the `_id`.
+
+  ![screen shot 2015-11-05 at 18 02 06](https://cloud.githubusercontent.com/assets/7833470/10987367/84e1849a-83e7-11e5-862b-5d03d0b119f1.png)
```

Comment:
are you going to add a screenshot for "Now get just one book using the _id."  (I think that should be step 2)